client/src/app/core/ui-services/login-data.service.ts

```diff
@@ -155,32 +155,33 @@ export class LoginDataService {
      */
     private async setup(): Promise<void> {
         await this.loadLoginData();
-        this.organisationSettingsService.get<string>('general_event_privacy_policy').subscribe(value => {
+        this.organisationSettingsService.get('privacy_policy').subscribe(value => {
             if (value !== undefined) {
                 this._privacyPolicy.next(value);
                 this.storeLoginDataRequests.next();
             }
         });
-        this.organisationSettingsService.get<string>('general_event_legal_notice').subscribe(value => {
+        this.organisationSettingsService.get('legal_notice').subscribe(value => {
             if (value !== undefined) {
                 this._legalNotice.next(value);
                 this.storeLoginDataRequests.next();
             }
         });
-        this.organisationSettingsService.get<string>('openslides_theme').subscribe(value => {
+        this.organisationSettingsService.get('theme').subscribe(value => {
             if (value) {
                 this._theme.next(value);
                 this.storeLoginDataRequests.next();
             }
         });
-        this.organisationSettingsService
-            .get<{ path: string; display_name: string }>('logo_web_header')
-            .subscribe(value => {
-                if (value) {
-                    this._logoWebHeader.next(value);
-                    this.storeLoginDataRequests.next();
-                }
-            });
+        // TODO!
```

Comment:
this is a resource now. The token is client-defined, so maybe create a constant somewhere.